### DISCUSSION TITLE
LSP banner appears and does not disappear even when changing lines.

### DISCUSSION THREAD

**Discussion by rafaelrcamargo** [helix-editor/helix#4036] [Q&A] [answered]
## Hey, I'm new using helix and have been loving it!
But one thing that's bothering me is these LSP flags that appear to describe a function, etc... But for me, those are showing and not disappearing until I press "Esc" which is a bit annoying, what can be a config or workaround for that?

The way VSCode, Neovim, etc... Do I think its more efficient, it shows up but when changing a line or writing it disappears.

![image](https://user-images.githubusercontent.com/66796237/193124381-44714400-07f7-4e21-8a03-1ae22a3dec17.png)
> Flag example

Thanks!

**Comment by the-mikedavis** [comment, marked as answer]
The popup can be closed with <kbd>Ctrl</kbd><kbd>c</kbd>. There's an issue for it getting stuck: https://github.com/helix-editor/helix/issues/3112. But it's intended to be open until exiting insert mode.

You can disable automatic signature-help popups with the `editor.lsp.auto-signature-help` option: https://docs.helix-editor.com/configuration.html#editorlsp-section

**Reply by rafaelrcamargo** [reply]
Got it, I have some key bindings for movement inside insert mode. The pop-up only closing after exiting insert mode explains the comportment.
`Esc` & `C-c` closes it successfully. But because of the motions it gets in the way, but by removing the `auto-signature-help` and just opening it manually using space mode resolves!
